certbot-apache/certbot_apache/parser.py

```diff
@@ -91,7 +90,12 @@ def __init__(self, root, vhostroot=None, version=(2, 4),
 
     def init_augeas(self):
         """ Initialize the actual Augeas instance """
-        import augeas
+
+        try:
+            import augeas
+        except ImportError:
```

Comment:
I personally think it's OK to just disable coverage here. Actually causing the `ImportError` during the tests is more trouble than it's worth I think.

Reply:
Oh yeah, I'm starting to remember...